packages/reports/addon/components/common-actions/schedule.hbs

```diff
@@ -2,212 +2,266 @@
+                        Data will be replaced in the same sheet every {{this.currentDisplayedRule.frequency}}
+                      {{else}}
+                        A new sheet will be created every {{this.currentDisplayedRule.frequency}}
+                      {{/if}}
+                    </p>
+                  {{/if}}
+                  <DenaliInputGroup @label="Status" class="m-b-30 align-items-center">
+                    <DenaliSwitch
+                      class="schedule__modal-disabled-status"
+                      @isEnabled={{not this.currentDisplayedRule.isDisabled}}
+                      @onChange={{toggle "isDisabled" this.currentDisplayedRule}}
+                      @onLabel="Active"
+                      @offLabel="Paused"
+                    />
+                  </DenaliInputGroup>
+                {{/if}}
+              {{/if}}
+              {{#if this.currentDisplayedRule}}
+                <div class="schedule__modal-buttons w-full p-24">
+                  {{#if (or (and this.currentDisplayedRule.isNew (not (await this.isValidForSchedule))) this.errorWhileFetchingRules)}}
+                    <DenaliButton
+                      class="schedule__modal-cancel-btn m-r-5"
+                      @style="ghost"
+                      disabled={{this.isSaving}}
+                      type="button"
+                      {{on "click" (queue this.closeModal this.revertAll)}}
+                    > Cancel
+                    </DenaliButton>
+                  {{else}}
+                    <DenaliButton
+                      class="schedule__modal-save-btn m-r-5"
+                      disabled={{this.disableSave}}
+                      isLoading={{this.isSaving}}
+                      {{on "click" (queue (toggle "isSaving" this) this.doSave)}}
+                    >
+                      Save
+                    </DenaliButton>
+                    <DenaliButton
+                      class="schedule__modal-cancel-btn m-r-5"
+                      disabled={{this.isSaving}}
+                      @style="ghost"
+                      type="button"
+                      {{on "click" (queue this.closeModal this.revertAll)}}
+                    >
+                      Cancel
+                    </DenaliButton>
+                  {{/if}}
+                  {{#unless this.currentDisplayedRule.isNew}}
+                    <CommonActions::Delete
+                      @model={{this.currentDisplayedRule}}
+                      @deleteAction={{queue (fn this.doDelete this.currentDisplayedRule) this.closeModal}}
+                      as |onDelete|
+                    >
+                      <DenaliButton
+                        disabled={{this.isSaving}}
+                        @style="text"
+                        @size="medium"
+                        class="schedule__modal-delete-btn"
+                        {{on "click" onDelete}}
+                      >
+                        Delete
```

Comment:
Was this in the spec? The delete behavior is different then the trash can on the left.